src/Tribe/Field.php

```diff
@@ -443,6 +443,12 @@ public function do_field() {
 					$tags['option']   = $common_attributes;
 					$tags['fieldset'] = _wp_add_global_attributes( [] );
 
+					// Allow the script and template tags for HTML fields (inserting script localization, js templates).
+					if ( $this->type === 'html' || $this->type === 'wrapped_html' ) {
+						$tags['script']   = _wp_add_global_attributes( [ 'type' => true ] );
```

Comment:
Note: `type` is needed for older script tags used as templates - prevents errors from HTML in script tags.

Reply:
When i need to compare more than one string I tend to use `in_array` because I won't have to repeat the variable. Makes for shorter conditionals the more strings we compare. Eg.

`in_array( $this->type, ['html', 'wrapped_html'] )`